src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -15,6 +16,172 @@
  */
 class TinyMCEConfig extends HTMLEditorConfig
 {
+    /**
+     * @config
+     * @var array
+     */
+    private static $tinymce_lang = [
```

Comment:
It seems like this static could only list the exceptions to the rule `strtok($input, '_')` and be a lot shorter.